### PR TITLE
fix: modify button and input styles on storybook

### PR DESCRIPTION
**Pull request by Lham42**
**This change:** (check at least one)

- [ ] Adds a new feature
- [ ] Fixes a bug
- [ ] Improves maintainability
- [x] Improves documentation
- [ ] Is a release activity

**Is this a breaking change?** (check one)

- [ ] Yes
- [x] No

**Is the:** (complete all)

- [x] Title of this pull request clear, concise, and indicative of the issue number it addresses, if any?
- [x] Test suite(s) passing?
- [x] Code coverage maximal?
- [x] Changeset added?

**What does this change address?**
Improves consistency in brand use and styles in storybook.

**How does this change work?**
- The pharos-button replaces generic button in checkbox group, combobox, radio group and select component
- Width adjusted for combobox and select components
- combobox form data section added to react storybooks

### CODE REVIEW

**Review by daneah — COMMENTED** [review]
I think this is also present in the TextArea and TextInput component stories

**Review by Niznikr — COMMENTED** [review]
Could you use the React versions of the components in the React stories and also include imports for the buttons you added?

**Review by Niznikr — COMMENTED** [review]
Could you remove the extra `{' '}` that got generated?

File: packages/pharos/src/components/combobox/PharosCombobox.react.stories.mdx [hunk 1]
```diff
@@ -171,3 +178,44 @@ export const Template = (args) => (
...
+          placeholder="Enter some text"
+          onChange={(e) => action('Change')(e.target.value)}
+          onInput={(e) => action('Input')(e.target['_input'].value)}
+          required
+        >
+          <span slot="label">Test me out</span>
+          <option value="1">Option 1</option>
+          <option value="2">Option 2</option>
+          <option value="3">Option 3</option>
+        </pharos-combobox>
```
Comment (Niznikr) [inline packages/pharos/src/components/combobox/PharosCombobox.react.stories.mdx:199]:
```suggestion
        <PharosCombobox
          name="my-combobox"
          placeholder="Enter some text"
          onChange={(e) => action('Change')(e.target.value)}
          onInput={(e) => action('Input')(e.target['_input'].value)}
          required
        >
          <span slot="label">Test me out</span>
          <option value="1">Option 1</option>
          <option value="2">Option 2</option>
          <option value="3">Option 3</option>
        </PharosCombobox>
```

File: .changeset/silly-hairs-explain.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+'@ithaka/pharos': patch
+---
+
+- Pharos button added to checkbox group. combobox, radio group and select in storybook
+- width modified for combobox and select in storybook
```
Comment (Niznikr) [inline .changeset/silly-hairs-explain.md:6]:
```suggestion
modify button and input styles in Storybook:

* add `<pharos-button> to checkbox group, combobox, radio group and select stories
* modify width for combobox and select in Storybook
```
We use the [imperative mood](https://grammar.collinsdictionary.com/us/easy-learning/the-imperative) for our changelog entries

**Review by daneah — APPROVED** [review]

File: .changeset/silly-hairs-explain.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+'@ithaka/pharos': patch
+---
+
+modify button and input styles in Storybook:
+
+* add `<pharos-button> to checkbox group, combobox, radio group and select stories
```
Comment (Niznikr) [inline .changeset/silly-hairs-explain.md:7]:
```suggestion
* add `<pharos-button>` to checkbox group, combobox, radio group and select stories
```

Comment (Niznikr) [reply .changeset/silly-hairs-explain.md:7]: Gave an incorrect suggestion 😅

**Review by Niznikr — APPROVED** [review]

**Pull request merged**